airflow/providers/airbyte/hooks/airbyte.py

```diff
@@ -188,11 +194,10 @@ def submit_sync_connection(self, connection_id: str) -> Any:
                 headers={"accept": "application/json"},
             )
         else:
-            conn = self.get_connection(self.http_conn_id)
             self.method = "POST"
             return self.run(
                 endpoint=f"{self.api_version}/jobs",
-                headers={"accept": "application/json", "authorization": f"Bearer {conn.password}"},
+                headers={"accept": "application/json", "authorization": f"Bearer {self.airbyte_connection}"},
```

Comment:
It's not an equivalent now. Also in other cases below it's the same
```suggestion
                headers={"accept": "application/json", "authorization": f"Bearer {self.airbyte_connection.password}"},
```

Reply:
good catch.